imports/constant/message.js

```diff
@@ -1,7 +1,8 @@
 //Task
 const TASK_CREATED = "Created task: ",
 	COMMENT_CREATED = "Created comment: ",
-	LOG_CREATED = "Created log: ";
+	LOG_CREATED = "Created log: ",
+	EMPLOYEE_CREATED = "Created employee whose name is: ";
```

Comment:
Please modified this to be consistent with other messages. In the callback, we only get ID, so we only put "Created xxx: ${_id}".